dns-controller/cmd/dns-controller/main.go

```diff
@@ -32,7 +32,8 @@ import (
 
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
-	_ "k8s.io/kubernetes/pkg/client/metrics/prometheus" // for client metric registration
+
+	// _ "k8s.io/kubernetes/pkg/client/metrics/prometheus" // for client metric registration
```

Comment:
I tracked down the PR that moved this package

https://github.com/kubernetes/kubernetes/pull/81432/files#diff-a9f93108ac59ff44885fc0aea2f29e71R29

and it looks like this can be updated to `k8s.io/component-base/metrics/prometheus/restclient`

Reply:
Thanks - great find!  Replaced with that new location